docs/src/modules/java/pages/agents.adoc

```diff
@@ -444,6 +444,20 @@ include::example$doc-snippets/src/main/java/com/example/application/RemoteMcpWea
 
 When using MCP endpoints in other Akka services, the service ACLs apply just like for xref:http-endpoints.adoc[HTTP endpoints] and xref:grpc-endpoints.adoc[gRPC endpoints].
 
+[#configuring_tool_call_limits]
+=== Configuring tool call limits
+
+Inside a single request/response cycle, an LLM can successively request the agent to call functions tools. After analyzing the result of a tool call, the LLM might decide to request another call to gather more context. The `akka.javasdk.agent.max-tool-call-steps` setting limits how many such steps may occur between a user request and the final AI response.
```

Comment:
maybe we should mention that is also valid for MCP tool calls.

Reply:
I think it may be fine to not explicitly mention, those are also tool calls.